content/en/docs/contribute/new-content/blogs-case-studies.md

```diff
@@ -157,6 +157,10 @@ To submit a blog post follow these directions:
     title: "Your Title Here"
     date: YYYY-MM-DD
     slug: text-for-URL-link-here-no-spaces
+    author: >
+      Author-1 (Affiliation),
+      Author-2 (Affiliation),
+      Author-3 (Affiliation)
     ---
     ```
 
```

Comment:
Aside: we could mention that when there are a mixture of authors with or without affiliations, we use “independent” (NB: not “Independent”) for the affiliation of authors not otherwise affiliated.

Reply:
That's a nice suggestion. I think we should put it like "Affiliation/independent" or suggest it in the bullet point above. What do you say?

Reply:
I think we should make it clear that you don't need to include an affiliation here, especially if we want to call out that "independent" shouldn't be capitalized. That feels like a small thing that can be missed or not known by reviewers.  

Looking at https://kubernetes.io/blog/2024/06/06/10-years-of-kubernetes/, should we also include a note that authors can link to their github? Are there acceptable links we'd like to include other than github?

Reply:
@a-mccarthy I think the GitHub link is the best to include as for other media like Twitter, the author may not have an account. About the affiliation part, If it's not mandatory, we can remove it and mention that you can put affiliation if you want and will give an example of usage.

Reply:
@arujjval, yes i think thats a good idea to have an example of affiliation if authors want to include it.

Reply:
BTW Blog article authors might not have a GitHub account (yes really)

Reply:
maybe we can avoid strict standards here. If the author has a GitHub account, we will mention GitHub. If Twitter, then that. If none of them, maybe something else like LinkedIn. In the end, it's just a way for readers to get to know more about the author.